teamcity/pytest_plugin.py

```diff
@@ -374,14 +374,33 @@ def pytest_terminal_summary(self):
 
     def _report_coverage(self):
         from coverage.misc import NotPython
-        from coverage.report import Reporter
         from coverage.results import Numbers
+        try:
+            from coverage.report import Reporter
+        except ImportError:
+            # Support for coverage >= 5.0.1.
+            from coverage.report import get_analysis_to_report
+
+            class Reporter(object):
```

Comment:
Could you create a proxy `class _Reporter(object)` for `coverage >= 5.0.1` and `coverage < 5.0.1`.
Than inherit the `_CoverageReporter` from it.

Reply:
Sure, I'll try.

As for tests - if I see it correctly, you already have tests for different coverage versions, so I hope it suffices to just enable latest coverage version.

Reply:
I think we can add coverage 5.0.1 here explicitly
https://github.com/JetBrains/teamcity-messages/blob/master/tests/integration-tests/pytest_integration_test.py#L168

Reply:
Well, yes. But with the latest version you will be notified that something has changed (not that I expect a fix soon...). I'd have both, 5.0.1 and latest then.